projects/cdk/services/static-request.service.ts

```diff
@@ -38,7 +38,7 @@ export class TuiStaticRequestService {
                 xhr.abort();
             };
         });
-        const piped = observable.pipe(shareReplay(1));
+        const piped = observable.pipe(shareReplay({bufferSize: 1, refCount: true}));
```

Comment:
I think this is intentional, it replays last value even after it was complete and then resubscribed, `refCount` will make it drop last value if nobody is subscribed.

Reply:
fixed